### DISCUSSION TITLE
Can't get html-head replacement to work

### DISCUSSION THREAD

**Discussion by dstrivelli** [tchiotludo/akhq#1415] [Q&A] [answered]
Between updates from 0.23.0 to the current Dev version, the html-head replacement appears to have stopped working.

<img width="675" alt="image" src="https://user-images.githubusercontent.com/31289764/213000859-97d9d67b-3866-4587-ba27-cf9f309d246a.png">

When I rollback to the 0.23.0 version, my defined snippet is replaced into the HTML head as expected.

**Comment by tchiotludo** [comment]
I've try right now, and it's working, can you share more detail please?

**Comment by michelsantos10** [comment]
I think it is because in html generated has the "/" on end and the replace in code does not see it.

generated html code (index.html): 
```html
<meta name="html-head" content="replace"/>
````
![image](https://github.com/tchiotludo/akhq/assets/20172046/63752e2e-7b30-42fe-966d-b430fb2dbd6c)


text checked to replace ([link source code](https://github.com/tchiotludo/akhq/blob/53af3a32db0df247920bafb5b85195151af34393/src/main/java/org/akhq/controllers/StaticFilter.java#L75C8-L75C122)):  
```java 
line = line.replace("<meta name=\"html-head\" content=\"replace\">", this.htmlHead == null ? "" : this.htmlHead);
```

**Comment by AlexisSouquiere** [comment]
Indeed, there was a bug in 0.24.0 but it has been fixed with #1489

**Reply by michelsantos10** [reply]
thank you for the feedback! Do you know when the fix will be available in a new version?

**Reply by tchiotludo** [reply, marked as answer]
see [here](https://github.com/tchiotludo/akhq/discussions/1644#discussioncomment-8356005)
